contracts/stellar-upgrader/src/tests/testdata/dummy-contract/src/contract.rs

```diff
@@ -1,19 +1,20 @@
 //! Dummy contract to test the `Upgrader` contract
 
-use soroban_sdk::{contract, contracterror, vec, Address, BytesN, Env};
 use stellar_axelar_std::interfaces::{OwnableInterface, UpgradableInterface};
-use stellar_axelar_std::{contractimpl, contractstorage, interfaces};
+use stellar_axelar_std::{
+    contract, contracterror, contractimpl, interfaces, soroban_sdk, vec, Address, BytesN, Env,
+};
 
 #[contract]
 pub struct DummyContract;
 
 #[contractimpl]
 impl UpgradableInterface for DummyContract {
-    fn version(env: &Env) -> soroban_sdk::String {
-        soroban_sdk::String::from_str(env, "0.1.0")
+    fn version(env: &Env) -> stellar_axelar_std::String {
```

Comment:
```suggestion
    fn version(env: &Env) -> String {
```
fully import it for readability?